x-pack/plugins/stack_connectors/public/connector_types/jira/jira.test.tsx

```diff
@@ -89,9 +89,7 @@ describe('jira action params validation', () => {
       errors: {
         'subActionParams.incident.summary': [],
         'subActionParams.incident.labels': [],
-        'subActionParams.incident.otherFields': [
-          'Additional fields field must be a valid JSON object.',
-        ],
+        'subActionParams.incident.otherFields': ['Invalid JSON.'],
```

Comment:
nit: It is not this PRs fault(maybe I caused it in the past?) but it annoys me **so much** that we call this `otherFields` in the code when in practice it should be `additionalFields` everywhere 😄